packages/atomic/src/components/result-link/result-link.tsx

```diff
@@ -12,18 +12,32 @@ export interface ResultLinkProps {
 export const LinkWithResultAnalytics: FunctionalComponent<ResultLinkProps> = (
   {href, interactiveResult, target, part},
   children
-) => (
-  <a
-    part={part}
-    href={filterProtocol(href)}
-    onClick={() => interactiveResult.select()}
-    onContextMenu={() => interactiveResult.select()}
-    onMouseDown={() => interactiveResult.select()}
-    onMouseUp={() => interactiveResult.select()}
-    onTouchStart={() => interactiveResult.beginDelayedSelect()}
-    onTouchEnd={() => interactiveResult.cancelPendingSelect()}
-    target={target}
-  >
-    {children}
-  </a>
-);
+) => {
+  const stopPropagationAndProcess = (e: Event, process: () => void) => {
+    e.stopPropagation();
```

Comment:
Curious, what is `stopPropagation` preventing in this case?

Reply:
If you have a link element within another link element, you don't want the click event to propagate in the DOM.

The click should only be handled once.